cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,130 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+cy.visit(
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+);
+cy.url().should(
+"contain",
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.visit('/actions/');
+  cy.url().should(
+    "contain", '/actions/'
+  );
+});
+
+When('the user clicks the "Add concepts" floating action button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+When('the user clicks the "Add concepts" MuiButton-label button', () => {
+  cy.findByText("Add concepts").click();
+});
+
+When('the user selects "Add bulk concepts"', () => {
+  cy.visit(
+    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+When(
+  /the user enters concept Id "(1000|1001|1002)"/, (conceptId) => {
+    cy.get("textarea").type("conceptId")
```

Comment:
We want to use the variable `conceptId` here, not the string "conceptId"

```suggestion
    cy.get("textarea").type(conceptId)
```